lib/std/builtin.zig

```diff
@@ -214,7 +214,7 @@ pub const TypeInfo = union(enum) {
         size: Size,
         is_const: bool,
         is_volatile: bool,
-        alignment: comptime_int,
+        alignment: u29,
```

Comment:
I think we should leave defining maximum alignment in ZIg as a part of #75. Stage1 uses `u29` since LLVM defines maximum alignment to be `1 << 29`.

Reply:
A u29 isn't big enough to fit 1<<29.  You need u30.

Reply:
I think a lot of things use u29 for alignment so they should all be changed together.

But, all of the ints in TypeInfo are intentionally comptime_int: https://github.com/ziglang/zig/issues/1683, so I'll change this and https://github.com/ziglang/zig/pull/6402 to comptime_int soon.